provisions/roles/openshift/tasks/setup_mailservice.yml

```diff
@@ -20,10 +20,25 @@
       chdir: "{{ ansible_env.HOME }}/cccp-service/mail_service"
 
 - name: Start Mail Service
+  when: production
   docker:
       name: mail-server
       state: restarted
       image: mail-server
       restart_policy: unless-stopped
       volumes:
         /srv/pipeline-logs:/srv/pipeline-logs:rw
+      env:
+        PRODUCTION: True
+
+- name: Start Mail Service
+  when: not production
+  docker:
+    name: mail-server
+    state: restarted
+    image: mail-server
+    restart_policy: unless-stopped
+    volumes:
+      /srv/pipeline-logs/:/srv/pipeline-logs:rw
+    env:
+      PRODUCTION: False
```

Comment:
Ansible allows boolean values for variables. Environment variables to be passed to container are bash environment variable, also the condition on which the env var value depends applies to sub operation of `Start Mail Service`. To avoid complexity, I simply put two different blocks.